### PR TITLE
update bootstrap and jQuery

### LINKED ISSUE #193: Update Bootstrap and jQuery
There are [known security vulnerabilities][lighthouse] in the versions we are currently using.

[lighthouse]: https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://die-lernwerkstatt.org

### PR DESCRIPTION
**Pull request by hikes101**
**What**:
- Update Bootstrap and jQuery

**Why**:
- Security vulnerabilities in previously used versions

**How**:

**Checklist**:

- [ ] Documentation added to the
      [README](https://github.com/Lernwerkstatt/site/blob/master/README.md) N/A
- [ ] Tests N/A
- [x] Ready to be merged
      

Fixes #193

### CODE REVIEW

File: src/views/layouts/main.hbs [hunk 1]
```diff
@@ -39,14 +39,13 @@
 
   <!-- Optional JavaScript -->
   <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
-    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+  <script src="http://code.jquery.com/jquery-3.4.1.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```
Comment (jarjan) [inline src/views/layouts/main.hbs]: We could use [unpkg](https://unpkg.com/) and don't think about it later.

<script src="https://unpkg.com/browse/jquery@^3/dist/jquery.js">

File: src/views/layouts/main.hbs [hunk 2]
```diff
@@ -39,14 +39,13 @@
...
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+    integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
```
Comment (jarjan) [inline src/views/layouts/main.hbs]: `<script src="https://unpkg.com/browse/popper.js@^1/dist/popper.min.js">`

File: src/views/layouts/main.hbs [hunk 3]
```diff
@@ -39,14 +39,13 @@
...
     crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"
-    integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49"
-    crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"
-    integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy"
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+    integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
```
Comment (jarjan) [inline src/views/layouts/main.hbs]: `<script src="https://unpkg.com/browse/bootstrap@^4/dist/js/bootstrap.min.js">`

File: src/views/layouts/main.hbs (hunk 1 above)
Comment (hikes101) [reply src/views/layouts/main.hbs]: Good idea, I changed it

**Pull request merged**